packages/cli/lib/errorHandlers/overrrideErrors.js

```diff
@@ -0,0 +1,95 @@
+const { isSpecifiedError } = require('@hubspot/local-dev-lib/errors/apiErrors');
```

Comment:
Looks like you snuck an extra "r" into override in the file name

Reply:
🤦‍♀️ Woops, thanks for catching that!